### PR TITLE
Rac 4181 apiv20 converts - nodes_tests.py, obm_settings.py, obm_tests.py

### PR DESCRIPTION
**Pull request by hohene**
One set of converted cit to fit scripts 
Converted scripts include:
removing proboscis from the scripts and updating to the unittest format
updating the style to meet flake8 checks
fixed scripts to operate properly against specific node types -compute, switch, pdu
fixed scripts to clean up added test nodes

These tests won't run in smoke until we move then to api-cit directory. 
@johren @larry-dean @brianparry @dalebremner @keedya @srinia6

### CODE REVIEW

File: test/tests/api/fit-staging/v2_0/nodes_tests.py [hunk 1]
```diff
@@ -106,33 +109,63 @@ def check_compute_count(self):
...
+        for n in nodes:
+            name = n.get('name')
+            if name in test_names:
+                uuid = n.get('id')
+                logs.info(' Deleting node %s (name=%s)', uuid, name)
+                Api().nodes_del_by_id(identifier=uuid)
+                rsp = self.__client.last_response
+                if rsp.status != 204:
+                    codes.append(rsp)
+                    logs.info(' Failed to delete node {} - {}'.format(uuid, name))
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/nodes_tests.py]: Wrong format for log message

File: test/tests/api/fit-staging/v2_0/obm_settings.py [hunk 2]
```diff
@@ -1,56 +1,67 @@
...
-            Api().nodes_get_catalog_source_by_id(uid,'rmm')
+            Api().nodes_get_catalog_source_by_id(uid, 'rmm')
             rsp = self.__client.last_response
             rmm = loads(rsp.data)
             if 'data' in rmm:
                 mac = bmc['data'].get('MAC Address')
         if mac is not None:
-            LOG.debug('BMC MAC {0} for {1}'.format(mac,uid))
+            logs.debug('BMC MAC {0} for {1}'.format(mac, uid))
+            logs.debug('BMC user {0} passowd {1}'.format(user, passwd))
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/obm_settings.py]: Wrong format for log message

File: test/tests/api/fit-staging/v2_0/obm_settings.py [hunk 3]
```diff
@@ -1,56 +1,67 @@
...
                 }
             }
-            LOG.info('Creating ipmi obm-settings for node {0} \n {1}'.format(uid,setting))
+            logs.info('Creating ipmi obm-settings for node %s : %s', uid, setting)
             try:
                 Api().obms_put(setting)
-            except rest.ApiException as e:
-                LOG.error(e)
+                rsp = self.__client.last_response
+                logs.debug(" IPMI OBM Set response {}".format(rsp.status))
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/obm_settings.py]: Wrong format for log message

File: test/tests/api/fit-staging/v2_0/nodes_tests.py [hunk 4]
```diff
@@ -157,93 +190,103 @@ def handle_graph_finish(self,body,message):
...
 
-    @test(groups=['delete-node-api2'], depends_on_groups=['patch-node-api2'])
+    # @test(groups=['delete-node-api2'], depends_on_groups=['patch-node-api2'])
+    @depends(after='test_node_patch')
     def test_node_delete(self):
-        """ Testing DELETE:/api/2.0/nodes/:id """
+        # Testing DELETE:/api/2.0/nodes/:id
         codes = []
-        test_names = []
+        # test_names = []    todo: this  is not used
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/nodes_tests.py]: remove

File: test/tests/api/fit-staging/v2_0/nodes_tests.py [hunk 5]
```diff
@@ -266,91 +310,102 @@ def test_node_catalogs_bysource(self):
...
 #                        done = True
-#                    except rest.ApiException as e:
+#                    except ApiException as e:
 #                        if e.status != 404:
 #                            raise e
 #                        timeout -= 1
-#                assert_not_equal(timeout, 0, message='Failed to delete an active workflow')
-#        assert_raises(rest.ApiException, Api().nodes_workflow_action_by_id, 'fooey', {'command': 'test'})
+#                self.assertNotEqual(timeout, 0, msg='Failed to delete an active workflow')
+#        self.assertRaises(ApiException, Api().nodes_workflow_action_by_id, 'fooey', {'command': 'test'})
```
Comment (larry-dean) [inline test/tests/api/fit-staging/v2_0/nodes_tests.py]: remove comment block

File: test/tests/api/fit-staging/v2_0/nodes_tests.py (hunk 1 above)
Comment (hohene) [inline test/tests/api/fit-staging/v2_0/nodes_tests.py]: done.

File: test/tests/api/fit-staging/v2_0/nodes_tests.py (hunk 4 above)
Comment (hohene) [inline test/tests/api/fit-staging/v2_0/nodes_tests.py]: done.

File: test/tests/api/fit-staging/v2_0/nodes_tests.py (hunk 5 above)
Comment (hohene) [inline test/tests/api/fit-staging/v2_0/nodes_tests.py]: removed the two commented out tests.

File: test/tests/api/fit-staging/v2_0/obm_settings.py (hunk 2 above)
Comment (hohene) [inline test/tests/api/fit-staging/v2_0/obm_settings.py]: fixed

File: test/tests/api/fit-staging/v2_0/obm_settings.py (hunk 3 above)
Comment (hohene) [inline test/tests/api/fit-staging/v2_0/obm_settings.py]: fixed.

**Comment by johren** [comment]
test this please

**Comment by johren** [comment]
test this please

**Comment by johren** [comment]
test this please

**Comment by johren** [comment]
test this please

**Pull request merged**